src/Microsoft.DotNet.Interactive.Formatting.Tests/HtmlFormatterTests.cs

```diff
@@ -61,7 +61,7 @@ public void Formatter_expands_properties_of_ExpandoObjects()
                 expando.Name = "socks";
                 expando.Count = 2;
 
-                var formatter = HtmlFormatter<ExpandoObject>.Create();
+                var formatter = HtmlFormatter<ExpandoObject>.GetBestFormatterFor();
```

Comment:
For a more intuitive signature and to align to the recent addition of `Formatter.Register<T>` as well as the non-generic signature, I'd suggest we do: `HtmlFormatter.GetBestFormatterFor<T>`.

Reply:
cool will do

Reply:
The following feel like they should not be public, though some tests use them

Would you prefer

1. they are public

2. they are internal and the tests use an IVT

3. I adjust the tests so there are not used directly (since they are really an implementation detail I think)

I think (3) is best?

Reply:
Agreed.